### DISCUSSION TITLE
Smaller distance beetwen elements

### DISCUSSION THREAD

**Discussion by jakubziebin** [Textualize/textual#3338] [Q&A] [answered]
I need to create a container similar to a table where rows will be generated, and the `height` of each row must be 1. I'd like to achieve a very small gap between the rows. Is there an elegant way to achieve this? I tried to create a `grid` with two rows (one for my element and one empty static) and used `grid-rows: 95% 5%;`, but it didn't work. In addition, there must be a `checkbox` in my line, but with a border, background and padding removed.

**Comment by davep** [comment, marked as answer]
Something like this?

```python
from textual.app import App, ComposeResult
from textual.containers import VerticalScroll
from textual.widgets import Checkbox

class CheckboxListApp(App[None]):

    CSS = """
    VerticalScroll Checkbox {
        border: none;
        margin: 0 0 1 0;
    }
    """

    def compose(self) -> ComposeResult:
        with VerticalScroll():
            for n in range(100):
                yield Checkbox(f"This is checkbox {n}")

if __name__ == "__main__":
    CheckboxListApp().run()
```

<img width="1045" alt="Screenshot 2023-09-18 at 10 04 17" src="https://github.com/Textualize/textual/assets/28237/05f498f0-d2cb-4e2a-a1de-39880e8ef648">

**Reply by jakubziebin** [reply]
Yes, exactly

**Reply by jakubziebin** [reply]
Ouh sorry, i did not mean that this is the answer to my question. I want to make the distance between the rows even smaller. Something like the design in figma: 
![image](https://github.com/Textualize/textual/assets/116113682/5eb0083d-8f44-4062-bbc9-78a5d59d3418)

**Reply by davep** [reply]
Textual applications run inside a terminal, you can't make a gap that's less than a character cell in height.
